main/glib/APKBUILD

```diff
@@ -10,7 +11,7 @@ license="LGPL-2.1-or-later"
 triggers="$pkgname.trigger=/usr/share/glib-2.0/schemas:/usr/lib/gio/modules"
 depends_dev="python3 gettext-dev bzip2-dev libxml2-utils libxslt
 	docbook-xml docbook-xsl"
-makedepends="$depends_dev pcre-dev meson ninja
+makedepends="$depends_dev pcre-dev meson dbus-dev dbus
```

Comment:
`dbus-dev` here will introduce a circular dependency since dbus needs `glib-dev` to build.

Reply:
Oh right, thanks for noticing

Reply:
why is it needed? it seems to build fine without dbus.

Reply:
Ah, it's for additional tests, but we can live without that to avoid the dependency loop.